### DISCUSSION TITLE
Full Screen mode should hide the bottom page bar.

### DISCUSSION THREAD

**Discussion by paw-bor** [jgraph/drawio-desktop#1663] [Ideas]
Full Screen mode should disabe (I mean hiden) pages bar at the bottom of window, i.e. the bookmarks bar that presents pages of opened documents. The bar takes a lot of place and I always hide it, especialy since it usually contains only one position - the page of current document. This feature would save me a few clicks - I use a feauture from top menu to hide it in order to obtain more workspace of the app.

**Comment by paw-bor** [comment]
Could someone explain to me why my proposition was given a "low quality" label?

**Reply by davidjgraph** [reply]
Was it original added as a issue, rather than a discussion?

**Reply by paw-bor** [reply]
I added it to the 'Ideas' category
